src/components/ratings/FutureProofRatingView.tsx

```diff
@@ -0,0 +1,56 @@
+import CircularRatingIndicator from '@components/ratings/CircularRatingIndicator'
+import React from 'react'
+import { SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native'
+import CertificatesList from '@components/ratings/CertificatesList'
+import RectangularRatingIndicator from '@components/ratings/RectangularRatingIndicator'
+import { DisplayableBusiness } from '@futureproof/typings'
+
+const FutureProofRatingView = ({ businessToDisplay } : { businessToDisplay : DisplayableBusiness }) => {
+  return (
+    <SafeAreaView style={styles.futureProofRatingViewStyle}>
+      <ScrollView>
+        <View style={styles.futureProofRatingTitleView}>
+          <CircularRatingIndicator circleWidth={150} circleHeight={150} progressBarWidth={14} progressValue={businessToDisplay.sustainabilityScore ?? 0} ratingName={'FutureProof'}/>
+          <CertificatesList/>
+        </View>
+        <Text style={styles.headingText}>BREAKDOWN</Text>
```

Comment:
This might also be able to be generalised to a map function applied to an array of objects containing a ratingName ad progressValue